app/views/pages/netlify_config.yaml.erb

```diff
@@ -26,3 +26,54 @@ collections:
         value_field: "{{slug}}"
         display_fields: ["title", "{{slug}}"]
       - {label: "Body", name: "body", widget: "markdown"}
+      - name: footer
```

Comment:
[question] What does this field do?

Reply:
I think that was left over from a previous version of this where I was experimenting with having one big "settings" file with footer as a little blob inside there. I'll remove. (I think there are probably workflow / git reasons why having global settings split across multiple files is preferable)

Reply:
Ah, nm, that is from a _different_ earlier experiment I was doing